tests/detect-pcre/detect-pcre-02/writepcap.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from scapy.all import *
+
+pkts = []
+
+load_layer("http")
+pkts += Ether(dst='ff:ff:ff:ff:ff:ff', src='00:01:02:03:04:05')/ \
+    Dot1Q(vlan=6)/ \
+    IPv6(dst='1.2.3.4', src='5.6.7.8')/TCP(sport=6666, dport=63, flags='P''A')/HTTP()/HTTPRequest(Method='GET', Path=' / ', Http_Version='HTTP/1.1', Host='www.emergingthreats.net', User_Agent='Mozilla/1.0', Content_Type='text/html; charset=utf-8\r\n\r\n15\r\n<!DOC')
+pkts += Ether(dst='ff:ff:ff:ff:ff:ff', src='00:01:02:03:04:05')/ \
+    Dot1Q(vlan=6)/ \
+    IPv6(dst='1.2.3.4', src='5.6.7.8')/TCP(sport=6666, dport=63, flags='P''A')/HTTP()/HTTPRequest(Content_Type='<TYPE html PUBLIC')
```

Comment:
Why is there a `<` here? Also, I do not think that `Content_Type` is the right key here..
It seems more like content that follows the type.. the beginning of an html page. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

Reply:
I was trying to create fragmented chunks. I think I'm doing it the wrong way as it doesn't generate the expected alert